apps/mark-scan/frontend/src/pages/ballot_invalidated_page.tsx

```diff
@@ -0,0 +1,44 @@
+/* istanbul ignore file - placeholder component that will change */
+import { useContext } from 'react';
+
+import { Screen, H1, Main, Button, Text } from '@votingworks/ui';
+
+import { confirmInvalidateBallot } from '../api';
+
+import { BallotContext } from '../contexts/ballot_context';
+import { ButtonFooter } from '../components/button_footer';
+
+export function BallotInvalidatedPage(): JSX.Element | null {
+  const { resetBallot, endVoterSession } = useContext(BallotContext);
+
+  const confirmInvalidateBallotMutation = confirmInvalidateBallot.useMutation();
+
+  function onPressContinue() {
+    confirmInvalidateBallotMutation.mutate(undefined, {
+      async onSuccess() {
+        resetBallot();
+        await endVoterSession();
+      },
```

Comment:
I noticed in the video that the screen briefly flashes to the "start voting" screen before going back to the pollworker screen, which seemed unintentional. Might have to do with the order of actions here, though I'm not sure.